scripts/qa/capture_model_row_counts.py

```
@@ -0,0 +1,391 @@
+
+def write_baseline(
+    conn: snowflake.connector.SnowflakeConnection,
+    label: str,
+    captured_at: datetime,
+    counts: list[ModelCount],
+) -> None:
+    cur = conn.cursor()
+    try:
+        cur.execute(CREATE_BASELINE_DDL)
+        cur.execute(
+            f"DELETE FROM {BASELINE_TABLE} WHERE label = %s",
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)